File: 920-uncommon-words-from-two-sentences/uncommon-words-from-two-sentences.java
class Solution {
    public String[] uncommonFromSentences(String s1, String s2) {
        Map<String, Integer> map = new HashMap<>();
        String[] str1 = s1.split(" ");
        String[] str2 = s2.split(" ");
        for (String word : str1) {
            map.put(word, map.getOrDefault(word, 0) + 1);
        }
        for (String word : str2) {
            map.put(word, map.getOrDefault(word, 0) + 1);
        }
        ArrayList<String> ans = new ArrayList<>();
        for (Map.Entry<String, Integer> mp : map.entrySet()) {
            if (mp.getValue() == 1) {
                ans.add(mp.getKey());
            }
        }

        return ans.toArray(new String[0]);

    }
}

File: 920-uncommon-words-from-two-sentences/uncommon-words-from-two-sentences.py
class Solution:
    def uncommonFromSentences(self, s1: str, s2: str) -> List[str]:
        s1_words = s1.split(' ')
        s2_words = s2.split(' ')
        total_words = s1_words + s2_words
        count = Counter(total_words)

        return [word for word in count if count[word] == 1]